test/plugin/test_out_rdkafka2.rb

```diff
@@ -116,5 +116,28 @@ def test_exclude_fields
       actual_messages = target_driver.events.collect { |event| event[2] }
       assert_equal([{'a' => 'b', 'message' => 'test'}], actual_messages)
     end
+
+    def test_max_enqueue_bytes_per_second
+      conf = config(default_topic: TOPIC_NAME) +
+             config_element('ROOT', '', {"max_enqueue_bytes_per_second" => 32 * 3,
+                                         "rdkafka_delivery_handle_poll_timeout" => 0}, [])
```

Comment:
how to conduct '32 * 3' as an appropriate bytes/second value?

Reply:
I'm adjusting it in order to finish this test in 2-3 seconds.
In this test, all messages are 32bytes, so it's limited to send 3 messages per second.
Since this test send 9 messages in total

* 3 messages will be sent in 0-1 second
* 3 (total 6) messages will be sent in 1-2 seconds
* 3 (total 9) messages will be sent just after above

So that total test time is expected in 2-3 seconds.

Reply:
(including staring & finishing the test driver)

Reply:
thank you for explanation, I got it.